app/subscriber/src/features/login/BrowserLogin.tsx

```diff
@@ -17,6 +19,17 @@ export const BrowserLogin: React.FC<IBrowserLoginProps> = ({ login }) => {
   const isLocal =
     new URL(authority).host.startsWith('localhost') ||
     new URL(authority).host.startsWith('host.docker.internal');
+
+  const [, api] = useAlerts();
+  const [alert, setAlert] = React.useState<IAlertModel>();
+
+  React.useEffect(() => {
+    api.findAllAlerts().then((data) => {
```

Comment:
Only fetch one record (the latest enabled message).  I know it will only have one record, but this implementation is not intuitive and will result in maintenance issues.